ftrr/src/main/java/framgia/co/edu/ftrr/controller/div/TraineeForRequestController.java

```diff
@@ -0,0 +1,26 @@
+package framgia.co.edu.ftrr.controller.div;
+
+import framgia.co.edu.ftrr.dto.request.InterviewDTO;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController("DivTraineeForRequestController")
+@RequestMapping("/edu/traineeforrequests")
+public class TraineeForRequestController extends DivController {
+    private static final Logger logger = LoggerFactory.getLogger(TraineeForRequestController.class);
+
+    @PostMapping("/{id}/interviewer")
```

Comment:
id ở đây là id của trainee for request phải ko em hì

em thêm đường dẫn vào là `requests/{request-id}/trainees/{trainee-for-resquest-id}/interviews` nhé

Reply:
mà em đang cho xử lý update thông tin interview cho từng trainee đúng hơm hì ;p

Reply:
đúng rồi chị. đây ko phải ở request controller thì e nghĩ đâu cần cho request id vào đâu chị